<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../lib/ItalcCore.qrc">
    <normaloff>:/resources/icon64.png</normaloff>:/resources/icon64.png</iconset>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QHBoxLayout" name="m_centralLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="m_scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="SideBar" name="m_sideBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="MainToolBar" name="m_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QToolBar</extends>
   <header>SideBar.h</header>
  </customwidget>
  <customwidget>
   <class>MainToolBar</class>
   <extends>QToolBar</extends>
   <header>MainToolBar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../italc.qrc"/>
  <include location="../../lib/ItalcCore.qrc"/>
 </resources>
 <connections/>
</ui>
